mosaic-runtime/src/commonTest/kotlin/com/jakewharton/mosaic/TestMosaicComposition.kt

```diff
@@ -0,0 +1,70 @@
+package com.jakewharton.mosaic
+
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import com.jakewharton.mosaic.ui.AnsiLevel
+import com.jakewharton.mosaic.ui.unit.IntSize
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.coroutineScope
+import kotlinx.coroutines.withTimeout
+
+private val DefaultTestTerminalSize = IntSize(80, 20)
+
+internal suspend fun runMosaicTest(
```

Comment:
Up next:
- Move this to mosaic-testing module
- Extract an interface for the control APIs + `setContent`.